README.md

```diff
@@ -145,6 +145,46 @@ Showing top 10 nodes out of 44 (cum >= 20ms)
 
 For Mac OSX golang profiling do not work.
 
+# Push as an App to Cloud Foundry
+
+1. Create doppler.firehose enabled user
+
+		uaac target https://uaa.[your cf system domain] --skip-ssl-validation
+		uaac token client get admin -s [your admin-secret]
+		uaac client add  [your doppler.firehose enabled user] --scope 'openid,oauth.approvals,doppler.firehose' --authorized_grant_types 'authorization_code,client_credentials,refresh_token' --authorities 'oauth.login,doppler.firehose' --secret [your doppler.firehose enabled user passwd]
```

Comment:
You need admin right to be able to add application name org, space name and so on.